### PR TITLE
Partially fixed LP Bug #1413983 with provided descriptions (5.6)

### PR DESCRIPTION
**Pull request by hrvojem**

### CODE REVIEW

File: doc/source/diagnostics/innodb_show_status.rst [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
 
-======================================
- Extended Show Engine |InnoDB| Status
-======================================
+====================================
+Extended Show Engine |InnoDB| Status
+====================================
 
-This feature reorganizes the output of ``SHOW ENGINE INNODB STATUS`` for a better readability and prints the amount of memory used by the internal hash tables. In addition, new variables are available to control the output.
+This feature reorganizes the output of ``SHOW ENGINE INNODB STATUS`` for a
```
Comment (laurynas-biveinis) [inline doc/source/diagnostics/innodb_show_status.rst]: s/a//

File: doc/source/diagnostics/innodb_show_status.rst [hunk 2]
```diff
@@ -328,62 +382,91 @@ The following variables contain information in the BUFFER POOL AND MEMORY sectio
...
 
 .. variable:: Innodb_read_views_memory
 
      :version 5.6.11-60.3: Introduced.
      :vartype: Numeric
      :scope: Global
 
-This status variable shows the total amount of memory allocated for the |InnoDB| read view (in bytes).
+This status variable shows the total amount of memory allocated for the
+|InnoDB| read view (in bytes).
```
Comment (laurynas-biveinis) [inline doc/source/diagnostics/innodb_show_status.rst]: s/view/views

File: doc/source/diagnostics/innodb_show_status.rst [hunk 3]
```diff
@@ -438,16 +535,18 @@ The following variables contain information in the TRANSACTIONS section of the o
...
 
-The following table contains information about the oldest active transaction in the system.
+The following table contains information about the oldest active transaction in
+the system.
 
 .. table:: INFORMATION_SCHEMA.XTRADB_READ_VIEW
 
    :column READ_VIEW_UNDO_NUMBER:
-   :column READ_VIEW_LOW_LIMIT_TRX_NUMBER: This is the highest transactions number at the time the view was created. 
+   :column READ_VIEW_LOW_LIMIT_TRX_NUMBER: This is the highest transactions number at the time the view was created.
```
Comment (laurynas-biveinis) [inline doc/source/diagnostics/innodb_show_status.rst]: s/transactions/transaction

**Review by laurynas-biveinis — COMMENTED** [review]
InnoDB parts LGTM with very minor comments addressed.
For TokuDB I'd replace memory allocator with "dynamic" or "heap" memory allocator, check with @georgelorchpercona

**Comment by george-lorch** [comment]
While "dynamic memory allocator" or "heap memory allocator" may be more accurate/pedantic definitions, I can not recall anywhere in written or verbal conversation where a memory allocation library such as the default in glibc, jemalloc, etc... was ever called anything other than just an "allocator, or "memory allocator", or "memory allocation library".

So maybe rewording maybe to something like:
"This variable shows the version of the memory allocator library detected by PerconaFT."
and
"Total number of bytes requested from memory allocator library."
and
"Total number of bytes allocated from the memory allocator library."
etc...

Or are we just splitting hairs?

**Comment by hrvojem** [comment]
fixed/pushed

File: doc/source/tokudb/tokudb_troubleshooting.rst [hunk 4]
```diff
@@ -1254,10 +1254,10 @@ The following is a reference of table status statements:
  Number of failed calls to ``realloc()``.
 
 ``memory: number of bytes requested``:
- Total number of bytes requested from mallocator.
+ Total number of bytes requested from memory allocator library.
 
 ``memory: number of bytes freed``:
- Total number of mallocated bytes freed (used - freed = bytes in use).
+ Total number of memory allocated bytes freed (used - freed = bytes in use).
```
Comment (george-lorch) [inline doc/source/tokudb/tokudb_troubleshooting.rst]: Missed one, maybe "Total number of bytes allocated from memory allocation library that have been freed (used - freed = bytes in use)."

**Comment by laurynas-biveinis** [comment]
We sure are splitting hairs. Still up to you @georgelorchpercona to approve TokuDB bits

**Review by laurynas-biveinis — APPROVED** [review]
InnoDB LGTM

**Review by george-lorch — APPROVED** [review]
Approved once the one remaining TokuDB freed description is touched.
